spec/controllers/alchemy/admin/elements_controller_spec.rb

```diff
@@ -2,10 +2,17 @@
 
 module Alchemy
   describe Admin::ElementsController do
-    let(:alchemy_page)         { create(:alchemy_page) }
-    let(:element)              { create(:alchemy_element, page_id: alchemy_page.id) }
-    let(:element_in_clipboard) { create(:alchemy_element, page_id: alchemy_page.id) }
-    let(:clipboard)            { session[:alchemy_clipboard] = {} }
+    let(:alchemy_page) { create(:alchemy_page) }
+
+    let(:element) do
+      create(:alchemy_element, page_version_id: alchemy_page.current_version_id)
```

Comment:
You can change `page_version_id: alchemy_page.current_version_id` to `page: page` in the factory instances! :)